samza-core/src/main/java/org/apache/samza/container/grouper/task/TaskPartitionAssignmentManager.java

```diff
@@ -50,20 +50,13 @@
   private final MetadataStore metadataStore;
 
   /**
-   * <ul>
-   * <li>
-   *   <p>
-   *     Builds the TaskPartitionAssignmentManager based upon the provided {@link MetadataStore} that is instantiated.
-   *     Setting up a metadata store instance is expensive which requires opening multiple connections
-   *     and reading a lot of information. Fully instantiated metadata store is taken as a constructor argument
-   *     to reuse it across different utility classes.
-   *   </p>
-   * </li>
-   *
-   * <li>
-   *   Uses the {@link CoordinatorStreamValueSerde} to serialize messages before reading/writing into metadata store.
-   * </li>
-   * </ul>
+   * <p>
```

Comment:
Do you need a <p> for a single paragraph? The other list item isn't relevant?

Reply:
Not necessary. Removed.